### ISSUE TITLE
Cannot find module '../node_modules/lodash/isArray.js'

### ISSUE THREAD

**Issue by David-Melo** [jamesfer/cypher-query-builder#68]
Hello, I've been using this great library for a while now. I recently tried to fork an existing app and can't get it to work. I'm receiving the following error. Thanks for your work on this library and thanks for taking time to review my issue.

`internal/modules/cjs/loader.js:582
    throw err;
    ^

Error: Cannot find module '../node_modules/lodash/isArray.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (C:\_\myappname\node_modules\cypher-query-builder\dist\cjs5.js:7:32)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)`

**Comment by jamesfer** [comment]
Hmm. Are you bundling the code with webpack? or another similar tool?

**Comment by David-Melo** [comment]
Hello, thanks for the speedy reply. Im using this on a simple express server and I'm pretty sure there's no webpack bundling or anything similar. Only way I was able to get it working was with downgrading to a previous version of this library (3.5.1).

Im not sure if some part of my setup is causing it to build incorrectly the relative path its looking for _.isArray is ../node_modules/ but from where it is located it should be at least one or two levels higher ../../. Im wondering if something in my setup is causing it to be confused at to the location? I'm not sure.

Like I said, I got it working with an older version since I was unable to progress any further.

I have attached a screenshot of the comparing the DIS folder between the latest version and the version i got working. 

![unnamed](https://user-images.githubusercontent.com/1429949/48498618-9f01ab00-e804-11e8-938a-521e740d64a3.jpg)
